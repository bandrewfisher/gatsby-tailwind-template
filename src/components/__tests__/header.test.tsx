import React from 'react';
import { render } from '@testing-library/react';
import Header from '../header';

describe('header test', () => {
  const createWrapper = () => render(<Header siteTitle="Hello" />);

  it('matches snapshot', () => {
    expect(createWrapper()).toMatchSnapshot();
  });
});
