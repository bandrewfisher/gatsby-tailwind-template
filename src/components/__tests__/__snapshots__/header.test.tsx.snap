// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header test matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        style="background: rebeccapurple; margin-bottom: 1.45rem;"
      >
        <div
          style="margin: 0px auto; max-width: 960px; padding: 1.45rem 1.0875rem;"
        >
          <h1
            style="margin: 0px;"
          >
            <a
              href="/"
              style="color: white; text-decoration: none;"
            >
              Hello
            </a>
          </h1>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      style="background: rebeccapurple; margin-bottom: 1.45rem;"
    >
      <div
        style="margin: 0px auto; max-width: 960px; padding: 1.45rem 1.0875rem;"
      >
        <h1
          style="margin: 0px;"
        >
          <a
            href="/"
            style="color: white; text-decoration: none;"
          >
            Hello
          </a>
        </h1>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
